py3status/modules/mpd_status.py

```diff
@@ -182,6 +183,8 @@ def attr_getter(attr):
                     return song_attr(song, attr)
 
                 text = self.py3.safe_format(self.format, attr_getter=attr_getter)
+                if isinstance(text, Composite):
+                    text = text.text()
```

Comment:
See `self.py3.get_composite_string`.

Reply:
Problem is that 

```python
        if not isinstance(format_string, Composite):
            return ""
```

will return empty string when the safe_format could return "invalid format" and thus would hide the error to the user :(

Reply:
Like many others, the better solution would be to deprecate `max_width` in favor of `format = '\?max_length=50 {name} [[[{artist} - ]{title}]|[{file}]]'`.

We fleshed out `get_composite_string` to deal with existing notification formats versus default notifications (which is often `None`) because it is more pretty than using `(default '')`... so it's really for `None` --> `''`. Maybe we can just flip the if statement... because we do more things for notifications to avoid sending notifications repeatedly.

Reply:
Regression commit: 6d99c33e7d11e43a3a463a1d53c0c1c85a135677. Easy thing to miss. ;-)